_tuning-your-cluster/cluster-manager-task-throttling.md

```diff
@@ -32,58 +32,65 @@ PUT _cluster/settings
   "persistent": {
     "cluster_manager.throttling.thresholds" : {
       "<task-type>" : {
-          "value" : <threshold limit>
+          "value" : <threshold>
       }
     }
   }
 }
 ```
 
-The following table describes the `cluster_manager.throttling.thresholds` object.
+The `cluster_manager.throttling.thresholds` object contains the following fields.
 
-Field Name | Description
+Field name | Description
 :--- | :---
-task-type | The task type. See [supported task types](#supported-task-types) for a list of valid values.
-value | The maximum number of tasks of the `task-type` type in the cluster manager's pending task queue. Default is `-1` (no task throttling).  
-
-## Supported task types
-
-The following task types are supported:
-
-- `create-index` 
-- `update-settings` 
-- `cluster-update-settings` 
-- `auto-create` 
-- `delete-index` 
-- `delete-dangling-index` 
-- `create-data-stream` 
-- `remove-data-stream` 
-- `rollover-index` 
-- `index-aliases` 
-- `put-mapping` 
-- `create-index-template` 
-- `remove-index-template` 
-- `create-component-template` 
-- `remove-component-template` 
-- `create-index-template-v2` 
-- `remove-index-template-v2` 
-- `put-pipeline` 
-- `delete-pipeline` 
-- `create-persistent-task` 
-- `finish-persistent-task` 
-- `remove-persistent-task` 
-- `update-task-state` 
-- `put-script` 
-- `delete-script` 
-- `put-repository` 
-- `delete-repository` 
-- `create-snapshot` 
-- `delete-snapshot` 
-- `update-snapshot-state` 
-- `restore-snapshot` 
-- `cluster-reroute-api`
-
-#### Example request
+`<task-type>` | The task type. For a list of valid task types, see [supported task types and default thresholds](#supported-task-types-and-default-thresholds).
+`<task-type>.value` | The maximum number of tasks of the `task-type` type in the cluster manager's pending task queue. <br> For default thresholds for each task type, see [Supported task types and default thresholds](#supported-task-types-and-default-thresholds).
+
+## Supported task types and default thresholds
+
+The following table lists all supported task types and their default throttling threshold values.
+
+Task type | Threshold
+:--- | :---
+`create-index `| 50
+`update-settings` | 50
+`cluster-update-settings` | 50
+`auto-create` | 200
+`delete-index` | 50
+`delete-dangling-index `| 50
+`create-data-stream` | 50
+`remove-data-stream` | 50
+`rollover-index` | 200
+`index-aliases` | 200
+`put-mapping` | 10000
```

Comment:
10,000?

Reply:
I think it's better not to have a comma because then people understand not to put a comma in the actual request. But doing code font for all numbers might be an overkill, so I"d leave as is.